dbt/main.py

```diff
@@ -414,6 +415,15 @@ def parse_args(args):
     )
     seed_sub.set_defaults(cls=seed_task.SeedTask, which='seed')
 
+    catalog_sub = subs.add_parser('catalog', parents=[base_subparser])
+    catalog_subs = catalog_sub.add_subparsers()
+    # it might look like catalog_sub is the correct parents entry, but that
+    # will cause weird errors about 'conflicting option strings'.
```

Comment:
@drewbanin should this be `docs`? `dbt catalog generate` vs `dbt docs generate`?

Reply:
Yeah, I was thinking `dbt docs generate` here. Might be worth changing the `GenerateTask` in `generate.py` to `DocsTask` and `docs.py`?

Reply:
Yup, I'll fix the catalog -> docs.

Is this actually a "docs task"? I assume the point of having `generate` be a subcommand of `docs` is that there are other `docs` tasks planned, right? If not, maybe this could just be `dbt docs`?